PhysicsTools/PatAlgos/python/slimming/lostTracks_cfi.py

```diff
@@ -1,5 +1,5 @@
 import FWCore.ParameterSet.Config as cms
-from PhysicsTools.PatAlgos.slimming.primaryVertexAssociation_cfi import primaryVertexAssociation
+from CommonTools.RecoAlgos.primaryVertexAssociation_cfi import primaryVertexAssociation
```

Comment:
Should this be all defined in the FillDescription of the  
PhysicsTools/PatAlgos/plugins/PATLostTracks.cc

Reply:
Right. Since PATLostTracks doesn't have fillDescriptions defined in the first place, perhaps this is a separate PR: https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatAlgos/plugins/PATLostTracks.cc